hazelcast/src/main/java/com/hazelcast/internal/util/StringUtil.java

```diff
@@ -531,4 +531,21 @@ public static String shorten(String s, int maxLength) {
         }
         return s.substring(maxLength - 1) + '…';
     }
+
+    /**
+     * Removes all occurrence of {@code charToRemove} from {@code str}.
```

Comment:
It would be useful to mention the motivation to have this method.

Reply:
Done.